napari/utils/_tests/test_notification_manager.py

```diff
@@ -75,8 +76,12 @@ def test_notification_manager_no_gui(monkeypatch):
 
         # pytest intercepts the error, so we can manually call sys.excepthook
         assert sys.excepthook == notification_manager.receive_error
-        sys.excepthook(*sys.exc_info())
+        try:
+            raise ValueError("a")
+        except ValueError:
+            sys.excepthook(*sys.exc_info())
```

Comment:
`sys.exc_info()` output was `(None, None, None)` that crash printing exception info and preventing to run all callbacks.